### ISSUE TITLE
4 |  MedianOfTwoSortedArrays | C++

### ISSUE THREAD

**Issue by gadm21** [SHY-Corp/LeetCode-Solutions#29]
## **`Title`** -

**What will change** -

### Type of Issue -

Please add/delete options that are not relevant.

- [x] Adding New Code
- [ ] Improving Code
- [ ] Improving Documentation
- [ ] Bug Fix

### Programming Language

Please add/delete options that are not relevant.

- [ ] Python
- [x] C++
- [ ] Java

### Self Check 

- Ask for issue assignment before making Pull Request.
- Add your file in the proper folder
- `Clean Code` and `Documentation` for better readability
- Add `Title` and `Description` of the program in the file

:star2: Star it :fork_and_knife:Fork it :handshake: Contribute to it!

Discord server ✉️ - https://discord.gg/Ftbj4a

Happy Coding,

**Comment by gadm21** [comment]
I want to work on this problem.

**Comment by Rohit-varma** [comment]
I want to work on this problem, can you assign me?

**Comment by Yashica7** [comment]
- [x]  Assigned to @gadm21

**Comment by Rohit-varma** [comment]
Can you assign me this issue, I have already created a pull request, please check that, my code takes case of all corner cases

**Comment by Yashica7** [comment]
@Rohit-varma Hi since this Issue has already been assigned to @gadm21 ,  you can create another Issue and wait for it to be assigned to you and only after that you can make a Pull Request
